app/Domain/Mail/MailManager.php

```diff
@@ -18,13 +18,15 @@ final class MailManager
     private const SUBJECT_INTEGRATION_ACTIVATED = 'Publiq platform - Integration activated';
     private const SUBJECT_INTEGRATION_BLOCKED = 'Publiq platform - Integration blocked';
     private const SUBJECT_INTEGRATION_CREATED = 'Welcome to Publiq platform - Let\'s get you started!';
+    private const SUBJECT_INTEGRATION_ACTIVATION_REMINDER_EMAIL = 'Publiq platform - Can we help you to activate your integration?';
 
     public function __construct(
         private readonly Mailer $mailer,
         private readonly IntegrationRepository $integrationRepository,
         private readonly int $templateIntegrationCreated,
         private readonly int $templateIntegrationActivated,
         private readonly int $templateIntegrationBlocked,
+        private readonly int $templateIntegrationActivationReminderEmail,
```

Comment:
I think this can move to a class that holds the template IDs, otherwise with every new template you need to change the signature of this method.

Reply:
I got exactly that idea, but I wanted to do it in a seperate pr if that is oké for you?

Reply:
Fine for me to do this in a separate pull request.